extensions/blocks/google-calendar/index.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createBlock } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import { extractAttributesFromIframe, URL_REGEX, IFRAME_REGEX } from './utils';
+import './editor.scss';
+import icon from './icon';
+
+export const name = 'google-calendar';
+export const title = __( 'Google Calendar', 'jetpack' );
+
+export const settings = {
+	title,
+	description: __( 'Embed a Google Calendar', 'jetpack' ),
+	keywords: [ __( 'calendar', 'jetpack' ), __( 'events', 'jetpack' ), __( 'dates', 'jetpack' ) ],
```

Comment:
Since "calendar" is already in the block name, it'll get discovered with that. 

Might be easier for translators to use this format to provide translation context: `_x( 'events', 'block search term', 'jetpack' )`